packages/stylelint-config/rules/stylelint-config-standard-scss.js

```diff
@@ -0,0 +1,42 @@
+module.exports = {
```

Comment:
I'm struggling with the file name a bit… Do you agree `stylelint-config-standard-scss-overrides.js` would describe its content better?

Reply:
❤️